### ISSUE TITLE
gulp and webpack?

### ISSUE THREAD

**Issue by Rajan** [trinile/graphql-mongodb-backend-example#2]
I think it can do without gulp if webpack is being used. isn't it?
Here's an example of webpack without gulp.
https://github.com/icebob/vue-express-mongo-boilerplate

**Comment by trinile** [comment]
True. I've seen with both and without. This was more to work with gulp a bit more than anything. Would do a version with just npm commands.
